packages/xo-server/src/xo-mixins/subjects.js

```diff
@@ -231,6 +244,68 @@ export default class {
     })
   }
 
+  async registerUser2(providerId, { user: { id, name }, data }) {
```

Comment:
It would be nice to add a small comment explaining this function?

And maybe what `user.id`, `user.name` and `data` are?